### DISCUSSION TITLE
What are the paramaters available to use on automaExecWorkflow()

### DISCUSSION THREAD

**Discussion by jddcef** [AutomaApp/automa#874] [Q&A] [answered]
If I want to call automaExecWorkflow() from javascript, what are the options /params names that I can pass to it?

**Comment by Kholid060** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/22908993/197083813-c8e4d1af-463c-4651-a91a-90d21d461293.png)

**Reply by jddcef** [reply]
Oh I see that's from https://docs.automa.site/blocks/create-element.html

The problem is google isn't indexing that docs site very well, it doesn't match on this query "site:https://docs.automa.site/ automaExecWorkflow"
